platform/src/main/java/org/hillview/storage/CassandraSSTableLoader.java

```diff
@@ -345,42 +295,42 @@ public ITable nonLazyLoading() {
                     i = 0;
                     /* Extracting the value for each column from a single row  */
                     for (ColumnData cd : row) {
-                        if (cd.column().isSimple()) {
-                            cell = (Cell) cd;
-                            cellType = cell.column().cellValueType();
-                            value = cellType.getSerializer().deserialize(cell.value());
-                        } else {
-                            throw new RuntimeException("[Hillview can't convert complex data]");
+                        /* Filter the column to only load the one marked true by columnToLoad[] */
+                        if (columnToLoad[i]){
+                            if (cd.column().isSimple()){
+                                cell = (Cell) cd;
+                                cellType = cell.column().cellValueType();
+                                value = cellType.getSerializer().deserialize(cell.value());
+                            }else{
+                                /* Hillview won't process the complex data */
```

Comment:
This is only a temporary solution, and probably you should throw this exception when you create the schema.
We should probably replace such columns with String values that use value.toString() as contents.